upgradeable-greeter/utils/transactionHelper.js

```diff
@@ -0,0 +1,22 @@
+const { calcEthereumTransactionParams } = require('@acala-network/eth-providers');
```

Comment:
I think we don't need this helper anymore, this way to calculate gas is more useful when we don't have a provider at hand. 

Since we already created a provider instance, using `provider._getEthGas(substrateParams)` is a short hand for `calcEthereumTransactionParams(substrateParams)`. `_getEthGas` takes care of `txFeePerGas` and `storageByteDeposit` automatically, so it's compatible with Acala in the future, which has different `storageByteDeposit`

Reply:
Yeah, the `transactionHelper` util is not used anymore. I'll remove it.